source/features/mark-merge-commits-in-list.tsx

```diff
@@ -33,8 +33,11 @@ const filterMergeCommits = async (commits: string[]): Promise<string[]> => {
 	return mergeCommits;
 };
 
-function getCommitHash(commit: HTMLElement): string {
-	return commit.dataset.channel!.split(':')[3];
+// eslint-disable-next-line import/prefer-default-export
+export function getCommitHash(commit: HTMLElement): string {
+	// Pre "Repository refresh" layout
+	return 	commit.dataset.channel!.split(':')[3] ||
```

Comment:
I think this should be `??`

Reply:
Channel still exists

Reply:
Your right. It returns undefined - I responded half in my sleep.
```suggestion
	return 	commit.dataset.channel!.split(':')[3] ??
```